mitiq/raw/raw.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2020 Unitary Fund
```

Comment:
This is a tangential point to the actual code, but from my experience, files that are newly created should be stamped with copyright pertaining to the year in which they are created. Similarly, any updates to a file with an older year that is updated in a more recent year should also have the copyright value updated to reflect this. 

Note that I am not a lawyer nor do I play one on the internet. This is just a practice I've had to do previously at other institutions and it seems to be somewhat consistent. Of course, it might be a better idea to confirm with a legal professional as opposed to an in-passing comment here from me.

Reply:
Good catch! Should be 2021, maybe even 2022 :upside_down_face:

Reply:
Updated here and in other files.